x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.tsx

```diff
@@ -787,26 +861,24 @@ export const defaultEndpointExceptionItems = (
         }),
       ];
     case 'ransomware':
-      return getProcessCodeSignature(alertEcsData).map((codeSignature) =>
+      return [
```

Comment:
Is there a reason to put the result of `getPrepopulatedRansomwareException` into array here and below? It seems od that we return an array with 1 element.

Reply:
I adjusted how we are calling `getPrepopulatedRanswomarWareException` to not use array.map so it no longer returns an array, which the original component that uses this return object expects.